packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -344,7 +344,7 @@ export interface AwsCustomResourceProps {
   /**
    * The memory size for the singleton Lambda function implementing this custom resource.
    *
-   * @default 512 mega in case if installLatestAwsSdk is false.
+   * @default 512 mega in case if installLatestAwsSdk is true.
```

Comment:
Looking at this implementation, the comments and the actual behavior seem to be reversed.

https://github.com/aws/aws-cdk/blob/4b6dc8c650822bcd0231c8890bd94a934a0cd34d/packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts#L486